GameofThrones2/app/src/main/java/com/example/gameofthrones/HouseList.java

```diff
@@ -0,0 +1,64 @@
+package com.example.gameofthrones;
+
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.recyclerview.widget.DividerItemDecoration;
+import androidx.recyclerview.widget.LinearLayoutManager;
+import androidx.recyclerview.widget.RecyclerView;
+
+import android.content.Intent;
+import android.graphics.Color;
+import android.os.Bundle;
+import android.widget.TextView;
+
+public class HouseList extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_house_list);
+        RecyclerView recyclerView = (RecyclerView)findViewById(R.id.CharacterList);
+        recyclerView.setLayoutManager(new LinearLayoutManager(this));
+
+        // Creating the arrays of the characters:
+        Person[] stark = new Person[11];
```

Comment:
Not sure if @ransh93 has taught ArrayList but a better solution would be to use an ArrayList which can grow as you need it and also supports the "add" method to add a new item to the end of the array.